tracing/src/main/java/com/palantir/tracing/Tracers.java

```diff
@@ -210,26 +210,40 @@ public static ScheduledExecutorService wrapWithNewTrace(String operation,
     /**
      * Deprecated.
      *
-     * @deprecated Use {@link #wrapWithNewTrace(String, Callable)}
+     * @deprecated Use {@link #wrapWithNewTrace(String, Callable, Optional)}
      */
     @Deprecated
     public static <V> Callable<V> wrapWithNewTrace(Callable<V> delegate) {
-        return wrapWithNewTrace(DEFAULT_ROOT_SPAN_OPERATION, delegate);
+        return wrapWithNewTrace(DEFAULT_ROOT_SPAN_OPERATION, delegate, Optional.empty());
+    }
+
+    /**
+     * Deprecated.
+     *
+     * @deprecated Use {@link #wrapWithNewTrace(String, Callable, Optional)}
+     */
+    @Deprecated
```

Comment:
@ferozco I actually disagree that this should be deprecated.  It appears in _so many places_ in our code, and I think forcing everyone to make the decision about whether it should be sampled will actually result in more people messing things up.

Reply:
Happy to keep it around then :)